bpf/src/cgnet.c

```diff
@@ -19,7 +19,7 @@
 #include <uapi/linux/ip.h>
 #include "bpf_helpers.h"
 
-struct bpf_map_def SEC("maps/count") count_map = {
+struct bpf_map_def SEC("maps/cgnet") count_map = {
```

Comment:
Please add a comment to document:
- what is the key: define the keys nearby the map definition, so it's clearer.
- what is the value: a counter
- how it is used to communicate: values initialized to zero by userspace and incremented by the ebpf program

Just before the map definition:
```
#define PACKETS_KEY 0
#define BYTES_KEY 1
```
In the count_packets function:
```
     int packets_key = PACKETS_KEY, bytes_key = BYTES_KEY;
```